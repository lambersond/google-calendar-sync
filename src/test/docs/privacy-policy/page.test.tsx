import { create } from 'react-test-renderer'
import PrivacyPolicy from '@/app/docs/privacy-policy/page'

describe('app/layout', () => {
  it('should match layout snapshot', () => {
    const tree = create(<PrivacyPolicy />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
