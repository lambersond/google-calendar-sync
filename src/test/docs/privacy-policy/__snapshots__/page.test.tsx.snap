// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app/layout should match layout snapshot 1`] = `
[
  <h1
    className="text-4xl font-bold text-center text-gray-200"
  >
    Google Calendar Sync
  </h1>,
  <div
    className="px-10 py-8"
  >
    <div
      className="flex flex-col w-full gap-6"
    >
      <section>
        <p
          className="text-4xl text-slate-200"
        >
          Privacy Policy
        </p>
        <li
          className="text-lg text-slate-100 pl-4"
        >
          Effective Date: 
          December 24, 2023
        </li>
        <li
          className="text-lg text-slate-100 pl-4"
        >
          Last Updated: 
          December 31, 2023
        </li>
      </section>
      <section>
        <p
          className="text-2xl text-slate-200"
        >
          Introduction
        </p>
        <li
          className="text-lg text-slate-100 pl-4"
        >
          This privacy policy (the "Policy") describes how Google Calendar Sync ("we," "us," or "our") collects, uses, and shares personal information when you use our single-page web application (the "App").
        </li>
      </section>
      <section>
        <p
          className="text-2xl text-slate-200"
        >
          Information We Collect
        </p>
        <li
          className="text-lg text-slate-100 pl-4"
        >
          Calendar Data: When you sync your calendars with the App, your events are stored locally in your browser's local storage.
        </li>
        <li
          className="text-lg text-slate-100 pl-4"
        >
          We do not store your calendar data on our servers.
        </li>
      </section>
      <section>
        <p
          className="text-2xl text-slate-200"
        >
          Your Information
        </p>
        <li
          className="text-lg text-slate-100 pl-4"
        >
          We do not share your calendar data with third parties.
        </li>
        <li
          className="text-lg text-slate-100 pl-4"
        >
          We do not sell your calendar data.
        </li>
      </section>
      <section>
        <p
          className="text-2xl text-slate-200"
        >
          Security
        </p>
        <li
          className="text-lg text-slate-100 pl-4"
        >
          We take security measures to protect your information. However, no security measures are perfect, and we cannot guarantee the security of your information.
        </li>
      </section>
      <section>
        <p
          className="text-2xl text-slate-200"
        >
          Changes to This Policy
        </p>
        <li
          className="text-lg text-slate-100 pl-4"
        >
          We may change this Policy from time to time. If we do, we will let you know by revising the date at the top of the policy.
        </li>
        <li
          className="text-lg text-slate-100 pl-4"
        >
          If we make a change to this policy that, in our sole discretion, is material, we will provide you with additional notice.
        </li>
        <li
          className="text-lg text-slate-100 pl-4"
        >
          We encourage you to review the Policy whenever you access the App or otherwise interact with us to stay informed about our information practices and the choices available to you.
        </li>
      </section>
      <section>
        <p
          className="text-2xl text-slate-200"
        >
          Contact Us
        </p>
        <li
          className="text-lg text-slate-100 pl-4"
        >
          If you have any questions, please contact us at
           
          <a
            className="italic underline hover:underline-offset-2"
            href="mailto:sync.calendar.webapp@gmail.com"
          >
            sync.calendar.webapp@gmail.com
          </a>
        </li>
      </section>
    </div>
  </div>,
]
`;
